### ISSUE TITLE
Error on startup: unhandled rejection

### ISSUE THREAD

**Issue by brnllski** [solana-labs/dapp-scaffold#79]
Error seen by me and another dev on discord, it happens when the app first starts and is loaded by the browser

Error is `Unhandled Rejection: failed to get info about account Invalid Param: Wrong Size`

I tracked it down to the getMultipleAccounts call in connection.tsx
Some of the known mint addresses had a trailing space in the address.

The below patch fixed it for me , but not sure if there is a better way to patch upstream of this to fix the addresses

```
diff --git a/src/contexts/connection.tsx b/src/contexts/connection.tsx
index 5d864e9..215eea5 100644
--- a/src/contexts/connection.tsx
+++ b/src/contexts/connection.tsx
@@ -105,7 +105,7 @@ export function ConnectionProvider({ children = undefined as any }) {
         .excludeByTag("nft")
         .getList();
       const knownMints = list.reduce((map, item) => {
-        map.set(item.address, item);
+        map.set(item.address.trim(), item);
         return map;
       }, new Map<string, TokenInfo>());
 
```

**Comment by kgangadharan** [comment]
Tried the fix on Mac. but did not help.

**Comment by kgangadharan** [comment]
trying to follow along on the https://github.com/pyth-network/pyth-examples, and got the same error.

**Comment by seanfcastillo** [comment]
On 12/17/21, I started getting this error as well, but for a different reason than OP, but as this is still Open, I will add my issue here.

`Unhandled Rejection (Error): failed to get info about account Invalid param: WrongSize`

For me, it appears to be a newly added token on Solana Smart Chain (added today). Related to the original post here, this is a token in the list of known mint addresses, but it is being rejected by one of the web3 APIs.

https://coinalpha.app/token/beeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeef
The contract id, 'beeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeef' is being rejected by the `Connection` class in `@solana/web3.js` when calling the `connection._rpcRequest("getMultipleAccounts", args);` method in accounts.tsx. I filtered the token out by name as a workaround, but this probably needs a fix in the web3 package.

in` connection.tsx` I modified the following line like so:
`  const accounts = await getMultipleAccounts(connection, [...knownMints.keys()].filter((x)=>isValidForConnectionClass(x)), 'single');`

and defined the following method:
`function isValidForConnectionClass(str) {
  if (str === 'beeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeef'){ return false; }
  return true;
}`
